### PR TITLE
Update author info and add acknowledgements

### PR DESCRIPTION
**Pull request by jjc2718**
Adding author info and acknowledgements.

Things that need to be checked/updated by @cgreene:
* Affiliations (wasn't sure if you want to include the Penn or CCDL ones)
* Funding information

I'm also not sure if these are the affiliations/info Brock and Maria want, but we can ask about that when we send it to them for comments.

### CODE REVIEW

File: content/metadata.yaml [hunk 1]
```diff
@@ -1,26 +1,48 @@
...
+    name: Casey S. Greene
+    initials: CSG
+    orcid: 0000-0001-8713-9213
+    twitter: GreeneScientist
+    email: casey@greenelab.com
+    affiliations:
+      - Department of Biochemistry and Molecular Genetics, University of Colorado School of Medicine, Aurora, CO, USA
+      - Center for Health AI, University of Colorado School of Medicine, Aurora, CO, USA
+    funders:
+      - Grant XXXXXXXX
```
Comment (cgreene) [inline content/metadata.yaml:48]:
```suggestion
      - National Institutes of Health's National Cancer Institute (R01 CA237170)
      - National Institutes of Health's National Human Genome Research Institute (R01 HG010067)
--


<br class="Apple-interchange-newline" style="caret-color: rgb(0, 0, 0); color: rgb(0, 0, 0); font-style: normal; font-variant-caps: normal; font-weight: normal; letter-spacing: normal; orphans: auto; text-align: start; text-indent: 0px; text-transform: none; white-space: normal; widows: auto; word-spacing: 0px; -webkit-text-size-adjust: auto; -webkit-text-stroke-width: 0px; text-decoration: none;">
```

File: content/metadata.yaml [hunk 2]
```diff
@@ -1,26 +1,48 @@
...
+    affiliations:
+      - Department of Computational and Systems Biology, School of Medicine, University of Pittsburgh, Pittsburgh, PA, USA
+    funders:
+      - Grant XXXXXXXX
+  - github: cgreene
+    name: Casey S. Greene
+    initials: CSG
+    orcid: 0000-0001-8713-9213
+    twitter: GreeneScientist
+    email: casey@greenelab.com
```
Comment (cgreene) [inline content/metadata.yaml:43]:
```suggestion
    email: casey.s.greene@cuanschutz.edu
```

**Review by cgreene — APPROVED** [review]
Works for me - we should email brock and maria for funding info or tag them on github 👍

File: content/metadata.yaml (hunk 1 above)
Comment (jjc2718) [reply content/metadata.yaml:48]: Did you want this bit of HTML in the last line here? It seems to break Manubot so I'm going to get rid of it for now, but let me know if it was intentional.

**Comment by jjc2718** [comment]
I'll plan to ask them when I send them a draft (not sure either of them are super active on here).

**Pull request merged**